src/Trixi.jl

```diff
@@ -37,12 +40,11 @@ using ConstructionBase: ConstructionBase
 @reexport using EllipsisNotation # ..
 using FillArrays: Ones, Zeros
 using ForwardDiff: ForwardDiff
-using HDF5: h5open, attributes
+using HDF5: h5open, attributes, create_dataset, datatype, dataspace, has_parallel
```

Comment:
```suggestion
using HDF5: h5open, attributes, create_dataset, datatype, dataspace
```
I would like to prefix `has_parallel` with `HDF5` to clarify that it's not something from us.

Reply:
Then you need to also import HDF5. Done in 7e29a2710fd1382ab2a80566e1d15f2952702eca.